lib/classes/TradeOffer.js

```diff
@@ -263,7 +262,7 @@ function addItem(details, offer, list) {
 		assetid: (details.assetid || details.id).toString(), // always needs to be a string
 		appid: parseInt(details.appid, 10), // always needs to be an int
 		contextid: details.contextid.toString(), // always needs to be a string
-		amount: parseInt(details.amount || 1, 10), // always needs to be an int
+		amount: parseInt(details.amount || 1, 10) // always needs to be an int
```

Comment:
We could create a function for it so it's still readable. I ran 1,000,000,000 iterations and timed both of the different methods to parse "4.3" and the bit shift method came out 39.4x faster
```
parseInt: 69899.18920898438ms
bit shift: 1775.699951171875ms
```